en/docs/reference/mediators/script-mediator.md

```diff
@@ -7,17 +7,9 @@ A Script mediator can be created in one of the following methods.
 -   With the script program statements stored in a separate file, referenced via the **Local or Remote Registry entry**.
```

Comment:
A Script mediator can be created in one of the following methods.
->
A Script mediator can be created using one of the following methods.